Simplenote/AccountRemote.swift

```diff
@@ -44,9 +38,7 @@ class AccountRemote: Remote {
     }
 
     private func deleteRequest(with user: SPUser) -> URLRequest? {
```

Comment:
Same here, we don't need to return an optional request here, we can change it to be non-optional.